### PR TITLE
🌐 Update FR translation for 1.4.0

### PR DESCRIPTION
**Pull request by ldidry**

### CODE REVIEW

File: assets/i18n/active.fr.json [hunk 1]
```diff
@@ -124,6 +156,22 @@
     "hash": "sha1-c915ad443fd65d4df5c1461ad85bc0ccc61b5654",
     "other": "Paramètre du sondage non reconnu : {{.Setting}}"
   },
+  "poll.newPoll.votesettings.invalidSetting": {
+    "hash": "sha1-a0649f6d7040b480c543e843e12cd14de97ef6d6",
+    "other": "Le nombre de votes doit être un nombre positif et inférieur ou égal au nombre d’options. Vous avez spécifié \"{{.MaxVotes}}\", mais le nombre d’options est \"{{.Options}}\""
```
Comment (hanzei) [inline assets/i18n/active.fr.json:161]: Would you please append a period at the end? I just added that to the source string in https://github.com/matterpoll/matterpoll/pull/331
```suggestion
    "other": "Le nombre de votes doit être un nombre positif et inférieur ou égal au nombre d’options. Vous avez spécifié \"{{.MaxVotes}}\", mais le nombre d’options est \"{{.Options}}\"."
```

**Review by hanzei — CHANGES_REQUESTED** [review]
Thanks for updating the translation :+1: One small request

File: assets/i18n/active.fr.json (hunk 1 above)
Comment (ldidry) [reply assets/i18n/active.fr.json]: Done :slightly_smiling_face:

**Review by kaakaa — APPROVED** [review]
👍

**Review by hanzei — APPROVED** [review]

**Comment by ldidry** [comment]
@hanzei Would you mind opting-in to hacktoberfest¹ and label this PR as `hacktoberfest-accepted`? Not that I created the PR for hacktoberfest, but since it’s october… :smirk: 

¹ new rules of hacktoberfest: https://go.digitalocean.com/index.php/email/emailWebview. You can see how to opt-in on https://www.youtube.com/watch?v=FxBv4FMcDgo&ab_channel=DigitalOcean (just adding `hacktober` to the topics of the repository)

**Comment by ldidry** [comment]
The link for the new rules isn’t right. This one is: <https://go.digitalocean.com/index.php/email/emailWebview?mkt_tok=eyJpIjoiWXpFNFltRXpPVFV4T1RjdyIsInQiOiJlU0RHQUt3TlBTWVZyYkxGQUR5ZHJBRXpBejZzendTNVN3UUdPaHRxSjNGOEJxWTFGMWxVTGJuZmNXUGwrdkwxTG5jYzRneW5KODlaam5CbjRSN0hMTUFCL0k5M1pRSE5TdDZpL2s5WVVobnNteTUxbjc1R0lidG9XSVBwMTY5SiJ9>

**Comment by hanzei** [comment]
@ldidry Sure, just added it. Thanks for reaching out :+1: 
Screenshot from 2020-10-06 16-16-03

**Comment by ldidry** [comment]
Thanks a lot :blush:

**Pull request merged**